apps/notifications/src/standalone/index.jsx

```diff
@@ -56,6 +56,8 @@ const customMiddleware = {
 			} ),
 	],
 	VIEW_SETTINGS: [ () => window.open( 'https://wordpress.com/me/notifications' ) ],
+	CLOSE_SHORTCUTS_POPOVER: [ () => sendMessage( { action: 'closeShortcutsPopover' } ) ],
+	TOGGLE_SHORTCUTS_POPOVER: [ () => sendMessage( { action: 'toggleShortcutsPopover' } ) ],
```

Comment:
Hi @eoigal :wave: I'm looking at this PR while reviewing #93766 and testing the cross-window messages that the `notifications` widget is exchanging with its parent window. And I'm wondering: why are these `closeShortcutsPopover` messages being sent? This `sendMessage` call is sending the message to the parent window but I don't see any handlers in the `wpcom` codebase.

The widget also has handlers for incoming `closeShortcutsPopover` -- see the new handlers that are added to `handleMessages` in this PR to the same file. But these are handlers for incoming messages from parent window. And again, I don't see the parent window in `wpcom` ever sending them.

So, what was the intent behind this?